Berksfile

```diff
@@ -3,7 +3,12 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'java'
-  cookbook 'minitest-handler'
-  cookbook 'stash_test', :path => 'test/cookbooks/stash_test'
+  cookbook 'java',
+    # Pending release: ark_java fails when tar is not installed
+    # See: https://github.com/agileorbit-cookbooks/java/issues/263
+    github: 'agileorbit-cookbooks/java'
```

Comment:
As I understand it, cookbooks that depend on this one never ever look at the Berksfile and its lockfile when doing dependency resolution -- just the metadata.rb. These are just used for us :)

The downside of locking is that our travis tests (once we get them going) might not immediately reveal a failure that I user might be seeing when an upstream cookbook breaks something -- we'd have to explicitly update the lockfile and commit it to see that failure. Either one of these approached could work for us